code/controllers/subsystem/icon_smooth.dm

```diff
@@ -19,8 +19,18 @@ SUBSYSTEM_DEF(icon_smooth)
 		can_fire = 0
 
 /datum/controller/subsystem/icon_smooth/Initialize()
-	smooth_zlevel(1,TRUE)
-	smooth_zlevel(2,TRUE)
```

Comment:
Curious why you changed this.

Reply:
We were already smoothing the entire world with the `/hook/startup` so this was just a redundant smooth. As for why removing it, in my opinion it makes sense to smooth the entire world at startup instead of each z-level manually.